projects/overlapping-long-subsessions.kp/knowledge.md

```diff
@@ -0,0 +1,588 @@
+      <td>3.703704</td>
+      <td>8.823529</td>
+      <td>16.666667</td>
+      <td>2289.0</td>
+      <td>66.666667</td>
+      <td>11.794920</td>
+      <td>0.043937</td>
+      <td>10.627409</td>
+    </tr>
+    <tr>
+      <th>2016-05</th>
+      <td>4.687500</td>
+      <td>11.111111</td>
+      <td>20.000000</td>
+      <td>3287.0</td>
+      <td>75.000000</td>
+      <td>13.292632</td>
+      <td>0.099404</td>
+      <td>11.029829</td>
+    </tr>
+    <tr>
+      <th>2016-06</th>
+      <td>6.034483</td>
+      <td>12.500000</td>
+      <td>20.000000</td>
+      <td>11633.0</td>
+      <td>75.000000</td>
+      <td>14.940370</td>
+      <td>0.026553</td>
+      <td>11.310856</td>
+    </tr>
+    <tr>
+      <th>2016-07</th>
+      <td>3.641907</td>
+      <td>8.333333</td>
+      <td>16.666667</td>
+      <td>2554.0</td>
+      <td>60.000000</td>
+      <td>11.196439</td>
+      <td>0.014749</td>
+      <td>9.827475</td>
+    </tr>
+    <tr>
+      <th>2016-08</th>
+      <td>4.494382</td>
+      <td>10.456439</td>
+      <td>18.666667</td>
+      <td>10970.0</td>
+      <td>66.666667</td>
+      <td>12.813731</td>
+      <td>0.016929</td>
+      <td>10.503366</td>
+    </tr>
+    <tr>
+      <th>2016-09</th>
+      <td>5.000000</td>
+      <td>11.111111</td>
+      <td>18.750000</td>
+      <td>7209.0</td>
+      <td>75.000000</td>
+      <td>13.160460</td>
+      <td>0.042265</td>
+      <td>10.601474</td>
+    </tr>
+    <tr>
+      <th>2016-10</th>
+      <td>6.451613</td>
+      <td>12.500000</td>
+      <td>20.000000</td>
+      <td>29266.0</td>
+      <td>75.000000</td>
+      <td>14.467175</td>
+      <td>0.041135</td>
+      <td>10.544938</td>
+    </tr>
+    <tr>
+      <th>2016-11</th>
+      <td>8.045977</td>
+      <td>14.814815</td>
+      <td>25.000000</td>
+      <td>20635.0</td>
+      <td>83.333333</td>
+      <td>16.648694</td>
+      <td>0.049529</td>
+      <td>11.189531</td>
+    </tr>
+    <tr>
+      <th>2016-12</th>
+      <td>3.225806</td>
+      <td>6.666667</td>
+      <td>11.864407</td>
+      <td>270810.0</td>
+      <td>75.000000</td>
+      <td>8.585782</td>
+      <td>0.013163</td>
+      <td>7.222534</td>
+    </tr>
+    <tr>
+      <th>2017-01</th>
+      <td>2.272727</td>
+      <td>5.000000</td>
+      <td>10.000000</td>
+      <td>323579.0</td>
+      <td>75.000000</td>
+      <td>7.345305</td>
+      <td>0.002613</td>
+      <td>7.132362</td>
+    </tr>
+    <tr>
+      <th>2017-02</th>
+      <td>8.333333</td>
+      <td>14.285714</td>
+      <td>20.000000</td>
+      <td>6413.0</td>
+      <td>75.000000</td>
+      <td>15.932386</td>
+      <td>0.049358</td>
+      <td>9.820090</td>
+    </tr>
+    <tr>
+      <th>2017-03</th>
+      <td>7.692308</td>
+      <td>14.285714</td>
+      <td>22.222222</td>
+      <td>72266.0</td>
+      <td>80.000000</td>
+      <td>16.511717</td>
+      <td>0.000919</td>
+      <td>11.530197</td>
+    </tr>
+  </tbody>
+</table>
+</div>
+
+
+
+### Using DocId
```

Comment:
There are no conclusions in prose. It would also be be good to have some hypothesis about what could be causing this problem. The current theory is that hibernation could be the culprit; is there any way to verify that?